src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -1537,6 +1537,54 @@ public override bool Equals(object obj)
 parameters: CSharp6);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
+        public async Task TestGenerateLiftedOperators()
```

Comment:
❓ Do we still have a test for a case where a value type `T?` does not use `==`?

Reply:
I don't see one, and it even appears that another test accidentally shows the opposite:

https://github.com/dotnet/roslyn/blob/bf98549d1b915d4e35fb52a6c873cb5287889651/src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs#L1047-L1066

I'll add one.

Reply:
I wrote a test and it is causing a stack overflow in the code I just wrote. 😁

Reply:
Done, added right below.